explainer.md

```diff
@@ -1122,16 +1122,16 @@ dictionary XRFrameOfReferenceOptions {
 //
 
 [SecureContext, Exposed=Window]
-interface XRVector3 {
```

Comment:
Maybe it's easier to define these as dictionaries.
A dictionary in WebIDL is just a JS object that is required to have certain members. 
It also doesn't introduce new classes in the js namespace so you don't need to get premission.

If you just make the change to dictionary, you will also allow the passing of XRVector and XRPoint to function that take a DOMPointInit

Reply:
Talking to our web API ergonomics contact here, I have been informed that returning dictionaries like this would definitely be "weird" from a web platform perspective. In _most_ cases, dictionaries are used to pass information into the web platform and interfaces are used to pass information back out.